<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Principal</class>
 <widget class="QMainWindow" name="Principal">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>982</width>
    <height>452</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="GLBox" name="openGLWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>800</width>
      <height>400</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_a">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>130</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>RED: 0</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_c">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>230</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>BLUE: 0</string>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider_c">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>250</y>
      <width>160</width>
      <height>20</height>
     </rect>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_HSV">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>80</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>HS&amp;V</string>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider_b">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>200</y>
      <width>160</width>
      <height>20</height>
     </rect>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_CMY">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>50</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>C&amp;MY</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_b">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>180</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>GREEN: 0</string>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider_a">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>150</y>
      <width>160</width>
      <height>20</height>
     </rect>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_RGB">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>20</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>R&amp;GB</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>0</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Escolha o modelo:</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>982</width>
     <height>27</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuPreenchimento_de_pol_gonos">
    <property name="title">
     <string>&amp;Cores</string>
    </property>
    <addaction name="actionCor_de_fundo"/>
   </widget>
   <addaction name="menuPreenchimento_de_pol_gonos"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionCor_da_linha">
   <property name="text">
    <string>&amp;Cor da linha</string>
   </property>
  </action>
  <action name="actionCor_de_fundo">
   <property name="text">
    <string>Cor &amp;de fundo</string>
   </property>
  </action>
  <action name="actionTabela_de_lados">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Tabela de lados</string>
   </property>
  </action>
  <action name="actionTabela_de_lados_ativos">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Tabela &amp;de lados ativos</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GLBox</class>
   <extends>QOpenGLWidget</extends>
   <header location="global">glbox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
